src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/StandardDeterminateSentence.kt

```diff
@@ -12,6 +12,7 @@ data class StandardDeterminateSentence(
   override val offence: Offence,
   val duration: Duration,
   override val sentencedAt: LocalDate,
+  val sentenceCalculationType: String,
```

Comment:
Might end up having to change a few tests :)

Reply:
Yes, this will involve changing a lot of JSON test files. Although I could make it default to Null, even though I think its always present.